py3status/modules/xrandr.py

```diff
@@ -384,6 +411,7 @@ def xrandr(self):
             full_text = self._center(self.displayed)
         else:
             full_text = self.displayed
+        full_text = self.py3.safe_format(self.format, {'xrandr': full_text})
```

Comment:
you changed xrandr -> output but not here

I'd also rename full_text to output when that is what it is being used for

so `full_text = self.py3.safe_format(self.format, {'output': output})`